### DISCUSSION TITLE
Welcome to Ecole_inclusive Discussions!

### DISCUSSION THREAD

**Discussion by DSMejantel** [DSMejantel/Ecole_inclusive#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ThierryM** [comment]
Adieu ;-),
Merci pour le partage : l'article dans Linuxfr.org m'a permis de découvrir SQLPage de façon concrète. Je suis ERUN dans l'Aude et ça faisait un moment que je cherchais un outil pour gérer des bases de données sur l'absentéisme mais sans avoir ni le temps, ni les compétences requises pour le réaliser.
Du coup, je me suis lancé dans l'installation sur un serveur perso que je relate ici : https://lofurol.fr/joomla/logiciels-libres/104-bases-de-donnees/349-sqlpage-utilisation-sur-un-nas-synology-avec-docker-et-mysql-postgresql.
Du coup, je vais tester ton application pour essayer d'en faire une dérivée pour une grosse école voulant un suivi pratique de toutes les aides mises en place pour les élèves "en difficulté".
Comme il s'agit d'un projet concernant l'Éducation nationale, pour avoir une meilleure exposition, que dirais-tu de l'héberger sur la nouvelle Forge mise en place dans les Apps Éducation  https://forge.apps.education.fr/ ?
Bien cordialement,

Thierry

**Comment by DSMejantel** [comment]
Bonjour Thierry,
Merci pour l'intérêt porté à ce projet.
Avec lovasoa, nous sommes en contact avec le "Chef de projet logiciels et ressources éducatives libres du Ministère de l'Éducation Nationale"
Le projet "École Inclusive" est disponible sur la forge des communs depuis lundi dernier.
Par rapport à ton projet, je pense que l'on peut l'utiliser quasiement en l'état. À partir du menu paramètres, il est possible de personnaliser les aides apportées aux élèves.
Je viens de mettre à jour le dépôt Github avec une fonctionnalité pour ajouter plus facilement les écoles depuis une recherche cartographique.
Bonne continuation

**Reply by ThierryM** [reply]
Merci pour les info.
Pour discuter, ça peut être intéressant d'utiliser les salons Tchap qui traitent des logiciels libres et de l'éducation. Alexis Kauffmann y est aussi.
J'avais parlé de École Inclusive et SQLPage sur le salon Tchap concernant la Forge justement : https://tchap.gouv.fr/#/room/!BXZZsyWklktciNEDbM:agent.education.tchap.gouv.fr/$X73ztWflSzbaNgZNbhW4T5GI2fxCagSo2NnssDwo8sI?via=agent.education.tchap.gouv.fr&via=agent.diplomatie.tchap.gouv.fr&via=agent.intradef.tchap.gouv.fr.
A lèu !
